toolbox/toolbox-test.sh

```diff
@@ -103,9 +105,13 @@ if [ -z "$(govc find / -type m -name "$vm")" ] ; then
   govc datastore.upload config.iso "$vm/config.iso" >/dev/null
   device=$(govc device.cdrom.add -vm "$vm")
   govc device.cdrom.insert -vm "$vm" -device "$device" "$vm/config.iso"
+
+  vm_path="$(govc find / -type m -name "$vm")"
+else
+  govc object.collect -s "$vm_path" -guest.toolsStatus toolsNot* # wait for previous toolbox to unregister
```

Comment:
Not that I know of.  The case above just prevents an annoying stream of error messages in the case of:

- toolbox{,.test} was started over ssh
- the toolbox process is killed but channel not closed
- no toolbox is running, but the vmx isn't sure yet so it won't accept a new channel
- toolbox keeps trying over and over (error message each time)
- vmx at some point gives up on the old toolbox channel and accepts the new, and everything is fine.